src/pages/starter-kit/create-integration.md

```diff
@@ -121,6 +121,10 @@ Use the following steps to create and activate an integration.
 
 If you are running Adobe Commerce 2.4.6 or higher, the modules that enable eventing are installed automatically. Skip to the next step. If you are running Commerce 2.4.4 or 2.4.5, you must install modules to enable eventing, as described in [Install Adobe I/O Events for Adobe Commerce](../events/installation.md).
 
+<InlineAlert variant="info" slots="text"/>
+
+By using an Adobe I/O Events for Commerce module version equal or greater to 1.6.0 you will benefit from some additional automated steps during the onboarding process.
```

Comment:
```suggestion
Using Adobe I/O Events for Commerce `1.6.0` or later provides some additional automated steps during the onboarding process.
```

@rojoangel Are these additional steps beneficial? Are we informing users of these additional steps for awareness purposes, or is there another reason?

Reply:
> Are these additional steps beneficial? 
They'll save some manual work: Commerce configuration will be done automatically for them. Is being more specific about the benefits worth it?

Reply:
@rojoangel i've updated my suggested change above, i think it sets expectations more clearly.